### DISCUSSION TITLE
Trigger value change after user stop typing

### DISCUSSION THREAD

**Discussion by GiggleLiu** [fonsp/Pluto.jl#946] [Questions about using Pluto]
I am trying to set some delay for the input elements to avoid being over reactive. But I do not know to the overwrite the listener in `stdlib.js`.

What I am trying to write:
```html
<script>
    var timer = '';
    const slider = document.getElementById("myid");
    slider.oninput = (e) => {
      e.preventDefault();
      clearTimeout(timer);
      timer = setTimeout(function() {
          # send the value to julia;
      }, delay);  // wait for delay (ms)
    }
</script>
```

Wondering what is the correct way of doing this.

using case:

For example, I have an number field that requires user to input a number. I find many people prefer deleting the number and then input a new one. However, when he delete the content, the number sent to Julia becomes `NaN`. Then, he will see an error message flashing.

**Comment by fonsp** [comment]
Hi @GiggleLiu 

It sounds like you are looking for a _"debouncer"_: https://github.com/fonsp/PlutoUI.jl/issues/69

Another possibility is _"throttling"_, you can read more about the difference here: https://css-tricks.com/the-difference-between-throttling-and-debouncing/
